packages/gatsby-image/src/index.js

```diff
@@ -249,6 +250,7 @@ class Image extends React.Component {
                 title={title}
                 src={image.base64}
                 style={imagePlaceholderStyle}
+                className={placeholderClass}
```

Comment:
Could we prep placeholder props early - because `alt`, `title`, `style` and now `className` seems to be repeated 4 times and we could prep 
```js
const placeholderImageProps = {
	title, 
	alt: !this.state.isVisible ? alt : ``},
	style: imagePlaceholderStyle,
	className: placeholderClass // or other prop name as I suggested
}
```

and then we could use something like
```js
<Img src={image.base64} {...placeholderImageProps} />
```